UPDATE_ME_NeosInstallLocation.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+   <PropertyGroup>
+      <!--Update this with the path to the directory where Neos.exe is-->
+      <NeosInstallDir>F:\Steam\steamapps\common\NeosVR</NeosInstallDir>
```

Comment:
This should probably be C:/whatever I forget, so it works by default if you've done nothing.

Reply:
This file is also a cheap hack, it may be better to auto generate this or something, so git doesn't try to version it (as it may be different per person, but nobody cares).

Reply:
The first comment here is fixed, it's still a cheap hack though.